cli/cmd/root.go

```diff
@@ -150,21 +173,66 @@ func init() {
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.vmclarity.yaml)")
-	rootCmd.PersistentFlags().StringVar(&output, "output", "", "set output directory path. Stdout is used if not set.")
+	rootCmd.PersistentFlags().StringVar(&output, "output", "", "set output directory path. Stdout is used if not set")
 	rootCmd.PersistentFlags().StringVar(&server, "server", "", "VMClarity server to export scan results to, for example: http://localhost:9999/api")
+	rootCmd.PersistentFlags().StringVar(&scanConfigName, "scan-config-name", "", "use an existing scan config that is defined in the VMClarity server")
 	rootCmd.PersistentFlags().StringVar(&scanResultID, "scan-result-id", "", "the ScanResult ID to export the scan results to")
 	rootCmd.PersistentFlags().BoolVar(&mountVolume, "mount-attached-volume", false, "discover for an attached volume and mount it before the scan")
 	rootCmd.PersistentFlags().BoolVar(&waitForServerAttached, "wait-for-server-attached", false, "wait for the VMClarity server to attach the volume")
+	rootCmd.PersistentFlags().StringVar(&input, "input", "", "input for families")
```

Comment:
The input can be a different type in the case of CICD mode. For example, it can be a directory and in the future we have to support container image, container info, and vm image input type. It makes sense only CICD mode because in orchestrated mode we are scanning VMs at the moment.

Reply:
IMO the input for all this should be the VMClarity CLI input yaml, ideally the CLI should have little to no command line arguments so that backwards compatibility is easier to maintain. For example the yaml could look something like:

```
asset:
  type: VM
  ...

loadFamilySettingsFromScanConfig: <ID or Name>
  
sbom:
  inputs:
    - type: ROOTFS
       path: /
```

If we want to provide the option to run VMClarity CLI without a yaml input, then IMO CLI flags we provide should be direct overrides of the options in the yaml "helm style" e.g. `--set asset.type=VM`

Reply:
agree, not sure why we need different cli flags for that mode